uber/templates/registration/check_in_form.html

```diff
@@ -0,0 +1,73 @@
+<table class="check-in">
```

Comment:
As you can see, this is just an HTML snippet rather than a full page.  Our ajax handler drops it into the dialog.